### DISCUSSION TITLE
GitHub Pages doesn't show style

### DISCUSSION THREAD

**Discussion by Hi-Pyncho** [community/community#22167] [Other Feature Feedback, Questions, & Ideas]
<p>Hello! I have a problem with Pages. It shows well my other projects, but with this project there is a situation.<br>
There is a link to Page:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://hi-pyncho.github.io/Vanilla-JS-Projects/Projects/tetris/index.html/#" class="site-icon" width="" height="">
      <a href="https://hi-pyncho.github.io/Vanilla-JS-Projects/Projects/tetris/index.html" target="_blank" rel="noopener nofollow ugc">hi-pyncho.github.io</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://hi-pyncho.github.io/Vanilla-JS-Projects/Projects/tetris/index.html" target="_blank" rel="noopener nofollow ugc">Tetris</a></h3>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>and to the code:<br>
</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/Hi-Pyncho/Vanilla-JS-Projects/tree/main/Projects/tetris" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars1.githubusercontent.com/u/58696550?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/Hi-Pyncho/Vanilla-JS-Projects/tree/main/Projects/tetris" target="_blank" rel="noopener nofollow ugc">Hi-Pyncho/Vanilla-JS-Projects</a></h3>

<p><a href="https://github.com/Hi-Pyncho/Vanilla-JS-Projects/tree/main/Projects/tetris" target="_blank" rel="noopener nofollow ugc">main/Projects/tetris</a></p>

  <p><span class="label1">:video_game: :computer: :monkey_face: Different projects using only JS, CSS, HTML. No libraries, no frameworks. </span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>It works locally nice. And i tried to create another repository and put the project in it. It also worked well.</p>
<p>I have no idea what is going on. Thanks for help!</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/hi-pyncho">@Hi-Pyncho</a>, welcome to the GitHub Support Community! I’m not quite sure what your issue is exactly. The page you’ve linked does appear to be styled using CSS from the <code>style.css</code> file that’s in your project’s directory.</p>
<p>Have you already managed to fix this or are there still issues? Can you post a screenshot of what you’re expecting to see?</p>

**Comment by Hi-Pyncho** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/7/47faf950abfeb6ffbd424392236a89f8d04bb3bf.png" data-download-href="/uploads/short-url/agLBMP2hrnFVAzLrqHdPxybbtTx.png?dl=1" title="error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/58696550/181069313-cc678548-8eaa-493b-974c-ecad9208ed1a.png" alt="error" data-base62-sha1="agLBMP2hrnFVAzLrqHdPxybbtTx" width="690" height="268" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47faf950abfeb6ffbd424392236a89f8d04bb3bf_2_690x268.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47faf950abfeb6ffbd424392236a89f8d04bb3bf_2_1035x402.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/7/47faf950abfeb6ffbd424392236a89f8d04bb3bf.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47faf950abfeb6ffbd424392236a89f8d04bb3bf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">error</span><span class="informations">1366×532 41.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment]
<p>I think I found the issue! It looks like you’re moving one level too high when trying to import your other CSS file:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Hi-Pyncho/Vanilla-JS-Projects/blob/3fb01677367d030ba549de9691a9ba04d076d947/Projects/tetris/style.css#L1" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Hi-Pyncho/Vanilla-JS-Projects/blob/3fb01677367d030ba549de9691a9ba04d076d947/Projects/tetris/style.css#L1" target="_blank" rel="noopener">Hi-Pyncho/Vanilla-JS-Projects/blob/3fb01677367d030ba549de9691a9ba04d076d947/Projects/tetris/style.css#L1</a></h4>
<pre class="onebox"><code class="lang-css"><ol class="start lines" start="1" style="counter-reset: li-counter 0 ;">
<li class="selected">@import '../../../basicStyle.css';</li>
<li>
<li>#tetris {</li>
<li>  border: 2px solid var(--light);</li>
<li>  margin: 0 auto 0 auto;</li>
<li>}</li>
<li>
<li>.title {</li>
<li>  color: var(--light);</li>
<li>  margin: 20px auto 30px auto;</li>
<li>  letter-spacing: 4px;</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>What you actually want is this:</p>
<pre><code class="lang-auto">@import '../../basicStyle.css';
</code></pre>

**Comment by otter-computer** [comment, marked as answer]
<p>As an addition to this, this is happening because GitHub Pages adds the repository name <code>Vanilla-JS-Projects</code> to the URL. This can be accounted for in local development when you build your site using Jekyll—GitHub’s default site builder—but likely won’t be when you’re using plain HTML or an alternate static site framework. Fixing this on GitHub Pages may break your development locally unless you account for this.</p>

**Comment by Hi-Pyncho** [comment]
<p>Thanks a lot! That helped!<br>
This worked locally on the computer so I didn’t notice it)</p>
